app/controllers/dashboard_controller.rb

```diff
@@ -7,6 +8,21 @@ def index
     @activities = Activity.all(current_user.token)
   end
 
+  def save_metrics_to_database
```

Comment:
Assignment Branch Condition size for save_metrics_to_database is too high. [15.03/15]